genesis/experiment/rootdomain/rootdomain.go

```diff
@@ -148,6 +150,17 @@ func (rd *RootDomain) DumpAllUsers() []byte {
 	return resJSON
 }
 
+func (rd *RootDomain) SetRoot(adminKey string) (string, *foundation.Error) {
```

Comment:
Should we pass adminKey as a part of constructor of RootDomain?

Reply:
We need publicKey to _create_ RootMember, so we can't create RootMember and save it as child in constructor of RootDomain.